packages/back-end/src/controllers/features.ts

```diff
@@ -520,20 +527,106 @@ export async function postFeatureRebase(
     status: 200,
   });
 }
+export async function postFeatureRequestReview(
+  req: AuthRequest<
+    {
+      comment: string;
+    },
+    { id: string; version: string }
+  >,
+  res: Response
+) {
+  const context = getContextFromReq(req);
+  const { id, version } = req.params;
+  const { comment } = req.body;
+  const feature = await getFeature(context, id);
+  if (!feature) {
+    throw new Error("Could not find feature");
+  }
```

Comment:
I think we want a permission check here.  Maybe the `createFeatureDrafts` permission?